controller_manager/src/controller_manager.cpp

```diff
@@ -881,9 +899,7 @@ int ControllerManager::RTControllerListWrapper::get_other_list(int index) const
 }
 
 void ControllerManager::RTControllerListWrapper::wait_until_rt_not_using(
-  int index,
-  std::chrono::microseconds sleep_period)
-const
+  int index, std::chrono::microseconds sleep_period) const
```

Comment:
I have the feeling that this should become a `rclcpp::Duration` type. Reason being is that I think this value would change when being run with simulated time.

Reply:
Agreed